### ISSUE TITLE
LinkRenderer always adding `rel="noopener noreferrer"` when opening in new window

### ISSUE THREAD

**Issue by joelclermont** [thephpleague/commonmark#992]
### Version(s) affected

2.4.0

### Description

When an external link is set to open in a new window, the `noopener` and `noreferrer` values are being added, even though my config has disabled them.

### How to reproduce

Here is my setup

```php
        $config = [
            'external_link' => [
                'internal_hosts' => ['my-internal-domain.com'],
                'open_in_new_window' => true,
                'nofollow' => '',
                'noopener' => '',
                'noreferrer' => '',
            ],
        ];

        $converter = new GithubFlavoredMarkdownConverter($config);
        $converter->getEnvironment()->addExtension(new ExternalLinkExtension());

        $converter->getEnvironment()->addRenderer(FencedCode::class, new FencedCodeRenderer());
        $converter->getEnvironment()->addRenderer(IndentedCode::class, new IndentedCodeRenderer());
```

Markdown input:
```
This is an external link [Google](https://google.com/).
```

HTML output:
```
<a target="_blank" href="https://google.com/" rel="noopener noreferrer">Google</a>
```

Expected output:
```
<a target="_blank" href="https://goggle.com/">Google</a>
```

**Comment by joelclermont** [comment]
And if I change the config for `open_in_new_window` to `false`, the `rel` attribute is not rendered (as expected).

**Comment by colinodell** [comment]
It looks like this is happening because `LinkRenderer` adds a default `rel` attribute if none is set:

https://github.com/thephpleague/commonmark/blob/564525f54da2824c99126fd517ce87934fd80a34/src/Extension/CommonMark/Renderer/Inline/LinkRenderer.php#L56-L58

Making `ExternalLinkProcessor` set the `rel` attribute to `false` should prevent this.  I'll get a PR together for this shortly :)

**Comment by colinodell** [comment]
Fixed in [v2.4.1](https://github.com/thephpleague/commonmark/releases/tag/2.4.1) :)

**Comment by joelclermont** [comment]
Beautiful!! Thank you!
